components/html-block/html-block.js

```diff
@@ -9,7 +9,13 @@ export const htmlBlockContentStyles = css`
 	.d2l-html-block-compact {
 		font-size: 0.8rem;
 		font-weight: 400;
-		line-height: 1.2rem;
+		line-height: 1.2;
+	}
+	.d2l-html-block-rendered > :first-child {
```

Comment:
This will have no impact if `no-deferred-rendering` is set, but in that case none of these styles apply anyways since we're not actually rendering anything into the component's shadowDOM.

Reply:
As an additional note, these selectors ignore text (or other non-element) nodes, so as can be seen in the demo, things can get weird if the first node isn't an element, because this CSS will happily strip the top margin from the first element node, even if some text node precedes it.